{{cookiecutter.project_slug}}/package.json

```diff
@@ -25,7 +24,6 @@
     "react-native-reanimated": "1.9.0",
     "react-native-screens": "1.0.0-alpha.22",
     "react-native-svg": "9.13.6",
```

Comment:
@tan-duong this library is needed because of `ui-kitten`, right? We could remove it if that's the only use of it

Reply:
No. it goes long with navigation and icon packages